### PR TITLE
refactor: refactor env var reading in docs/conf.py

### PR DESCRIPTION
**Pull request by ai-naymul**
Centralized the environment variable in the conf.py, refactor the code in constants.py

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -3,6 +3,11 @@
 import sys
 from os import path
 
+version = ''
+# Centralize environment variables
+JINA_VERSION = os.environ.get('JINA_VERSION')
+SPHINX_MULTIVERSION_VERSION = os.environ.get('SPHINX_MULTIVERSION_VERSION', version)
```
Comment (JoanFM) [inline docs/conf.py:9]: this is not what is intended. See that `version` is used later, and thus this reading should be placed after the `version` var initialization

File: docs/conf.py [hunk 2]
```diff
@@ -3,6 +3,11 @@
 import sys
 from os import path
 
+version = ''
+# Centralize environment variables
+JINA_VERSION = os.environ.get('JINA_VERSION')
```
Comment (JoanFM) [inline docs/conf.py:8]:
```suggestion
JINA_VERSION = os.environ.get('JINA_VERSION', None)
```

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: docs/conf.py (hunk 1 above)
Comment (ai-naymul) [reply docs/conf.py:9]: Okay I let me remove that...

File: docs/conf.py [hunk 3]
```diff
@@ -3,10 +3,10 @@
 import sys
 from os import path
 
-version = ''
+
 # Centralize environment variables
 JINA_VERSION = os.environ.get('JINA_VERSION')
-SPHINX_MULTIVERSION_VERSION = os.environ.get('SPHINX_MULTIVERSION_VERSION', version)
+SPHINX_MULTIVERSION_VERSION = os.environ.get('SPHINX_MULTIVERSION_VERSION', None)
```
Comment (JoanFM) [inline docs/conf.py:9]: no, this is not correct, this assignment needs to go much lowrr, aftrr the version variable is properly set

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: docs/conf.py (hunk 3 above)
Comment (ai-naymul) [reply docs/conf.py:9]: > no, this is not correct, this assignment needs to go much lowrr, aftrr the version variable is properly set

Oh. I am extremely sorry I didn't notice that...

Comment (ai-naymul) [reply docs/conf.py:9]: Is that fine?
kolla

Comment (JoanFM) [reply docs/conf.py:9]: no. Follow the flow, in master code we have:

```python
os.environ.get("SPHINX_MULTIVERSION_VERSION", version) on line 170
```
and on line 31 we have:

```python
version = __version__ 
```

With your changes, the default would be empty, in master the default would be the value of `__version__`. So ur code does not refactor, but changes the flow of the code

Comment (ai-naymul) [reply docs/conf.py:9]: > no. Follow the flow, in master code we have:
> 
> ```python
> os.environ.get("SPHINX_MULTIVERSION_VERSION", version) on line 170
> ```
> 
> and on line 31 we have:
> 
> ```python
> version = __version__ 
> ```
> 
> With your changes, the default would be empty, in master the default would be the value of `__version__`. So ur code does not refactor, but changes the flow of the code


Should I add a empty `version` string at the begining?
Screenshot_6

Comment (JoanFM) [reply docs/conf.py:9]: no, there is no need. But you cannot do this 

`os.environ.get("SPHINX_MULTIVERSION_VERSION", version)`

So early in the code

Comment (JoanFM) [reply docs/conf.py:9]: the code 

```python
SPHINX_MULTIVERSION_VERSION  = os.environ.get('SPHINX_MULTIVERSION_VERSION', version)
```
needs to be after line 31

Comment (JoanFM) [reply docs/conf.py:9]: yes

Comment (ai-naymul) [reply docs/conf.py]: > 

Ok

**Review by JoanFM — APPROVED** [review]

**Comment by JoanFM** [comment]
Thanks for the contribution!!

**Pull request merged**